gpt4/long.py:60: error: Item "list[list[float]]" of "ndarray[Any, Any] | DataFrame | list[list[float]]" has no attribute "T"  [union-attr]
gpt4/long.py:76: error: Incompatible types in assignment (expression has type "ndarray[Any, dtype[Any]]", variable has type "None")  [assignment]
gpt4/long.py:77: error: Incompatible types in assignment (expression has type "ndarray[Any, dtype[Any]]", variable has type "None")  [assignment]
gpt4/long.py:79: error: No overload variant of "cumsum" matches argument type "None"  [call-overload]
gpt4/long.py:79: note: Possible overload variants:
gpt4/long.py:79: note:     def [_SCT <: generic] cumsum(a: _SupportsArray[dtype[_SCT]] | _NestedSequence[_SupportsArray[dtype[_SCT]]], axis: SupportsIndex | None = ..., dtype: None = ..., out: None = ...) -> ndarray[Any, dtype[_SCT]]
gpt4/long.py:79: note:     def cumsum(a: Buffer | _SupportsArray[dtype[Any]] | _NestedSequence[_SupportsArray[dtype[Any]]] | bool | int | float | complex | str | bytes | _NestedSequence[bool | int | float | complex | str | bytes], axis: SupportsIndex | None = ..., dtype: None = ..., out: None = ...) -> ndarray[Any, dtype[Any]]
gpt4/long.py:79: note:     def [_SCT <: generic] cumsum(a: Buffer | _SupportsArray[dtype[Any]] | _NestedSequence[_SupportsArray[dtype[Any]]] | bool | int | float | complex | str | bytes | _NestedSequence[bool | int | float | complex | str | bytes], axis: SupportsIndex | None = ..., dtype: dtype[_SCT] | type[_SCT] | _SupportsDType[dtype[_SCT]] = ..., out: None = ...) -> ndarray[Any, dtype[_SCT]]
gpt4/long.py:79: note:     def cumsum(a: Buffer | _SupportsArray[dtype[Any]] | _NestedSequence[_SupportsArray[dtype[Any]]] | bool | int | float | complex | str | bytes | _NestedSequence[bool | int | float | complex | str | bytes], axis: SupportsIndex | None = ..., dtype: dtype[Any] | type[Any] | _SupportsDType[dtype[Any]] | str | tuple[Any, int] | tuple[Any, SupportsIndex | Sequence[SupportsIndex]] | list[Any] | _DTypeDict | tuple[Any, Any] | None = ..., out: None = ...) -> ndarray[Any, dtype[Any]]
gpt4/long.py:79: note:     def [_ArrayType <: ndarray[Any, dtype[Any]]] cumsum(a: Buffer | _SupportsArray[dtype[Any]] | _NestedSequence[_SupportsArray[dtype[Any]]] | bool | int | float | complex | str | bytes | _NestedSequence[bool | int | float | complex | str | bytes], axis: SupportsIndex | None = ..., dtype: dtype[Any] | type[Any] | _SupportsDType[dtype[Any]] | str | tuple[Any, int] | tuple[Any, SupportsIndex | Sequence[SupportsIndex]] | list[Any] | _DTypeDict | tuple[Any, Any] | None = ..., out: _ArrayType = ...) -> _ArrayType
gpt4/long.py:109: error: Value of type "None" is not indexable  [index]
Found 5 errors in 1 file (checked 1 source file)
